src/Native/CMakeLists.txt

```diff
@@ -143,7 +143,15 @@ if (NOT WIN32)
         endif()
     else (CMAKE_SYSTEM_NAME STREQUAL Darwin)
         # Ensure that objcopy is present
-        find_program(OBJCOPY objcopy)
+        if(DEFINED ENV{CROSSCOMPILE})
+            if(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)
+                find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
+            elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64)
+                find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
```

Comment:
Could you use an `OR` here instead of repeating the body, or do you think the `armv71` and `aarch64` tool names will diverge soon?